sktime/networks/resnet.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""Residual Network (ResNet) (minus the final output layer)."""
+
+__author__ = ["James Large", "Withington", "nilesh05apr"]
+
+from sktime.networks.base import BaseDeepNetwork
+from sktime.utils.validation._dependencies import (
+    _check_dl_dependencies,
+    _check_soft_dependencies,
+)
+
+_check_soft_dependencies(
```

Comment:
Any reason to have this here? I don't think you're making use of the import anywhere. This estimator can run without having `keras-self-attention` as a soft-dep.